tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -21,9 +21,9 @@
 
 @pytest.mark.regions(["us-east-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.oss(["centos7", "alinux"])
+@pytest.mark.oss(["centos7", "alinux", "alinux2", "ubuntu1604", "ubuntu1804"])
 @pytest.mark.schedulers(["sge"])
-@pytest.mark.usefixtures("os", "instance", "scheduler")
+@pytest.mark.usefixtures("instance", "scheduler")
```

Comment:
Why did you remove "os"?

Reply:
It isn't needed when one of the other OS markers is used. It wasn't hurting anything I just figured I'd remove unneeded markers if I was changing the file anyway.